addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
I was referring to RenderCapture only because I thought you meant a compatibility layer for that. Maybe I misunderstood. As mentioned, I dropped the API bump here. Discussion about how to have backwards compatibility should continue on the forum http://forum.kodi.tv/showthread.php?tid=179203&pid=2190022#pid2190022

> And a compatibility layer system is in general not a bad idea, is it?

In general ok but not always possible. This API broke due to changes in core. I cleaned some hacky globals and don't intend to create some hack in a compatibility layer to get part of that crap back.